zerver/lib/email_mirror.py

```diff
@@ -104,8 +105,8 @@ def create_missed_message_address(user_profile, message):
     # type: (UserProfile, Message) -> Text
     if settings.EMAIL_GATEWAY_PATTERN == '':
         logging.warning("EMAIL_GATEWAY_PATTERN is an empty string, using "
-                        "NOREPLY_EMAIL_ADDRESS in the 'from' field.")
-        return settings.NOREPLY_EMAIL_ADDRESS.split()[-1]
+                        "FromAddress.NOREPLY in the 'from' field.")
+        return FromAddress.NOREPLY
```

Comment:
I think we want this to still refer back to the sysadmin-facing setting, `NOREPLY_EMAIL_ADDRESS`, since this is a sysadmin-facing warning.